src/parsers/power_system_table_data.jl

```diff
@@ -760,6 +762,12 @@ function make_thermal_generator(data::PowerSystemTableData, gen, cost_colnames,
         var_cost = [(0.0, var_cost[1][2]), (1.0, var_cost[1][2])]
         fixed = 0.0
     end
+    return var_cost, fixed
+end
+
+function make_thermal_generator(data::PowerSystemTableData, gen, cost_colnames, bus)
+    fuel_cost = gen.fuel_price / 1000
```

Comment:
Why division over 1000. Also, don't divide over an integer.

Reply:
You are computing fuel_cost twice. Perhaps return it from `calculate_variable_cost`?